src/runtime/ranger/ranger_resource_policy_manager.cpp

```diff
@@ -97,5 +158,11 @@ ranger_resource_policy_manager::ranger_resource_policy_manager(
                               "RPC_CM_RENAME_APP"},
                              _ac_type_of_database_rpcs);
 }
+
+void parse_policies_from_json_for_test(const rapidjson::Value &data,
+                                       std::vector<policy_item> &policies)
+{
+    parse_policies_from_json(data, policies);
```

Comment:
It's a bit of strenge, how about define `parse_policies_from_json` as a static private function instead?

Reply:
Yes, when serializing like `access_type::KRead | access_type::KWrite`, will fail